importer/indico_importer/static/js/importer.js

```diff
@@ -492,6 +491,90 @@
                 }
             },
 
+            _getUrl: function(eventId, day, destination) {
+                var params = "?" + $.param({'day': day});
```

Comment:
no need to build a query string manually - just like `url_for` in python `build_url` will add any arguments not consumed while building the url path to the query string